### PR TITLE
AzurestackStorage SDK update for container migration and unsupported feature removal

### PR DESCRIPTION
**Pull request by cheguv**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as a link to the swagger spec, used to generate the code.
- [x] The `project.json` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@cheguv__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (cheguv). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ResourceManagement/AzureStackStorageAdmin/StorageManagement/StorageManagement.csproj [hunk 1]
```diff
@@ -14,7 +14,7 @@
   <Import Project="..\..\..\..\tools\Library.Settings.targets" />
   <ItemGroup>
     <Compile Include="Customization\EventQueryException.cs" />
-    <Compile Include="Customization\StorageAdminManagementClient.Customization.cs" />
+    <!-- Compile Include="Customization\StorageAdminManagementClient.Customization.cs" /> -->
     <Compile Include="Generated\StorageAdminManagementClient.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
```
Comment (bganapa) [inline src/ResourceManagement/AzureStackStorageAdmin/StorageManagement/StorageManagement.csproj]: please update the assembly version and SDK nuget package version

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

File: src/ResourceManagement/AzureStackStorageAdmin/StorageManagement/StorageManagement.csproj (hunk 1 above)
Comment (cheguv) [inline src/ResourceManagement/AzureStackStorageAdmin/StorageManagement/StorageManagement.csproj]: Fixed and change pushed.

**Comment by shahabhijeet** [comment]
@eridai @cheguv requested review from you. Once you approve this can be merged.

**Pull request merged**